src/utils/index.js

```diff
@@ -22,3 +23,21 @@ export function partition(array, isValid) {
     [[], []]
   );
 }
+
+/**
+ * Lists sub commands inside a code block
+ *
+ * @param {Array.<any} subCommands - sub commands to be listed
```

Comment:
missing the ending bracket, also no new line above param pls

Reply:
cool will do